docs/30_structural_analysis/includes/autopick/README.md

```diff
@@ -2,6 +2,6 @@
 
 The system can automatically pick peaks of interest for a specific range of wavenumbers with a defined noise level using `Auto peak picking` tab.
 
-Auto peak picking is based on the Savitzky Golay filter.
+Auto peak picking is based on the [Savitzky Golay filter](https://en.wikipedia.org/wiki/Savitzky%E2%80%93Golay_filter).
```

Comment:
hmm .. you cannot peak pick by just smoothing

Reply:
The gist is 

>is searching for inflection points to determine the position of peaks and the width of the peaks are between the 2 inflection points. The result of GSD yield to an array of object containing {x, y and width}. However this width is based on the inflection point and may be different from the 'fwhm' (Full Width Half Maximum).

The second algorithm (optimizePeaks) will optimize the width as a FWHM to match the original peak. After optimization the width with therefore be always FWHM whichever is the function used.

Reply:
https://github.com/mljs/global-spectral-deconvolution